tests/test_mem_eff_attention.py

```diff
@@ -823,3 +826,45 @@ def test_cuda_streams(
         atol=op.FORWARD_ERROR_ATOL[dtype],
         rtol=op.FORWARD_ERROR_RTOL.get(dtype, 1e-5),
     )
+
+
+@cuda_only
+@pytest.mark.parametrize("seed", [42])
+@pytest.mark.parametrize("k_len", [32])
+@pytest.mark.parametrize("batch_size", [2])
+@pytest.mark.parametrize("kv_len", [32])
+@pytest.mark.parametrize("q_len", [33])
+@pytest.mark.parametrize("device", ["cuda"])
+def test_custom_scale(device, q_len, kv_len, batch_size, k_len, seed):
+    pre_scale = 3
+    query = torch.randn((batch_size, q_len, k_len), device=device) * pre_scale
+    key = torch.randn((batch_size, kv_len, k_len), device=device) * pre_scale
+    value = torch.randn((batch_size, kv_len, k_len), device=device) * pre_scale
+    grad_out = torch.ones_like(query)
+
+    query.requires_grad_(True)
+    key.requires_grad_(True)
+    value.requires_grad_(True)
+
+    attn_bias = None
+    p = 0.0
+    scale = 2.0
+
```

Comment:
I believe filtering out unsupported operators will be needed for CI to work properly
```suggestion
    dispatch = xformers.ops.AttentionOpDispatch.from_arguments(
        query=query, key=key, value=value, attn_bias=attn_bias, scale=scale
    )
    if not op.supports(dispatch):
         pytest.skip(f"{op.NAME}: unsupported ({dispatch})")
```